integreat_cms/release_notes/current/unreleased/2252.yml

```
@@ -0,0 +1,2 @@
+en: Fix not editable problem of URL parameter field
+de: Behebung des Problems, dass das URL-Parameterfeld nicht editierbar war
```

```suggestion
en: Fix the problem of the URL parameter field not being editable
de: Behebe das Problem, dass das URL-Parameterfeld nicht editierbar ist
```
Similar to git commit messages, we use the imperative mood for our release notes.